data/benefits/javascript/caf_gironde-do-not-review-merge-aide-au-bafa-pour-une-session-de-formation-dapprofondissement.yml

```diff
@@ -0,0 +1,31 @@
+label: (DO NOT REVIEW/MERGE) Aide au BAFA pour une session de formation
```

Comment:
Il faut changer le nom du fichier `caf_gironde-do-not-review-merge-aide-au-bafa-pour-une-session-de-formation-dapprofondissement.yml` en `data/benefits/javascript/caf_gironde-aide-au-bafa-pour-une-session-de-formation-dapprofondissement.yml`

Reply:
Hello @Valandr, peux tu me dire comment faire ça vu que @Kout95 est parti ? Je n'ai pas trouvé ici. Merci